libs/editor/build.gradle

```diff
@@ -70,17 +70,17 @@ dependencies {
 
     // Forcing version due to https://github.com/facebook/SoLoader/issues/94
     // To be removed with React Native 0.70+
-    implementation("com.facebook.soloader:soloader") {
+    implementation("com.facebook.soloader:soloader:$facebookSoloaderVersion") {
```

Comment:
I noticed that the version wasn't being provided before the change here. All CI jobs are succeeding so seems it's ok to add it, however, I wanted to mention it just in case.

Reply:
Yes, you are right @fluiddot , this change isn't affecting anything, just making this dependency more robust! 💯 

FYI: As per [this commit](https://github.com/wordpress-mobile/WordPress-Android/pull/18543/commits/57c922d529d33eb2696423f0a069845c35df2ab7) and its description, I specified the reason behind this change, let me know if that still makes sense.

Reply:
> FYI: As per [this commit](https://github.com/wordpress-mobile/WordPress-Android/pull/18543/commits/57c922d529d33eb2696423f0a069845c35df2ab7) and its description, I specified the reason behind this change, let me know if that still makes sense.

Yeah, I overlooked the commit description but clearly explains the reason. Thanks!

Reply:
Awesome, thanks for the reply and checking the commit description @fluiddot , I am glad it still makes sense! ❤️ 😅